### ISSUE TITLE
Plugins are not enabled/disabled when handling multiple plugins at once

### ISSUE THREAD

**Issue by TheGarkine** [twilio-infra-as-code/twilio-pulumi-provider#15]
This is a bug I am currently having: When I try to create/delete multiple plugins at once, the do not get enabled deleted. I think this has to do with pulumis' parallelism, I'll check if there is some way around that!

**Comment by TheGarkine** [comment]
Made a workaround with the `dependsOn` option. This only workds for creation, though... This also causes all plugins to be deployed after each other, so its rather slow especially if you have multiple plugins. I have to check the documentation of ``twilio flex:plugins:release` to find a better solution.

```javascript
let previous_plugin = undefined

for (let plugin_dir of flexPlugins) {
    let dir = FLEX_PLUGIN_DIRECTORY + plugin_dir;
    let pack = require(dir + "/package.json");
    let options = {}
    if (previous_plugin != undefined) {
        options = { dependsOn: [previous_plugin] }
    }
    let plugin = new FlexPlugin(
        pack.name,
        { 
            attributes: {
                cwd: dir,
                release: true,
            }
        },
        options
    );
    previous_plugin = plugin
}
```

**Comment by kaiquelupo** [comment]
Just to make sure we are on the same page @TheGarkine , even with the last merges to master, this is still a problem, right?

**Comment by TheGarkine** [comment]
Yes, my guess is, that this is a result of multiple plugins trying to release at once. They probably have some sort of config which they load at the beginning of the release looking like this

```
 plugin1: {released: False},
 plugin2: {released: False},
 plugin3: {released: False},
```
And then they upload the whole thing setting their own released flag to true, overwriting changes that other might have done in the meantime (basically a race condition).

I could look into the flex-plugin-builder (https://github.com/twilio/flex-plugin-builder/tree/main/packages/plugin-flex), to see what actually happens, but my project is already delayed enough, maybe I find the time soon. 

We could also separate the deploy and release steps into different resources, such that we only would have to run the release synchronously.

**Comment by kaiquelupo** [comment]
Yep, it makes sense @TheGarkine. So I am thinking that the best course of action is to release right now the new package version and leave this issue for the next version. If you agree with that, I can publish the package today
